consensus/src/consensus.rs

```diff
@@ -98,14 +98,10 @@ impl ConsensusState {
         );
 
         let min_round = last_committed_round.saturating_sub(gc_depth);
-        let cert_map = cert_store
-            .iter(Some(Box::new(move |(_dig, cert)| {
-                cert.header.round > min_round
-            })))
-            .await;
+        let cert_map = cert_store.after_round(min_round + 1).unwrap();
```

Comment:
Maybe add a comment saying "get all >= min_round"?

Reply:
<pedantic> That's actually a > </pedantic>